src/points/saga.ts

```diff
@@ -170,7 +171,10 @@ export function* getPointsConfig() {
   }
 
   yield* put(getPointsConfigStarted())
+  yield* fetchPointsConfig()
```

Comment:
i think that we would need to put `yield* put(getPointsConfigStarted())` inside the `fetchPointsConfig` saga...the `pointsDataRefreshStarted` action currently doesn't trigger state changes, so if for whatever reason the config CF hangs, the points home screen (which listens to `pointsConfigStatus`) won't display the loader. having said that, why do we need `fetchPointsConfig` as a separate abstraction? why not just use `getPointsConfig`?

Reply:
or maybe `pointsDataRefreshStarted` should update the states to loading?

Reply:
@kathaypacific great question!
i extracted the config fetching part exactly to not display the config loader.
mainly because this particular config update is used in pull to refresh, which has its own UI to display the loading state.

my way of thinking is that the config loader makes sense until we get some config. because if we can't get any config, we can't proceed with the points at all. and we do display that loader when we request config for the first time on the [home screen mount](https://github.com/valora-inc/wallet/blob/5b451c73b1d61309bae2ca12df2f219903ffaa5e/src/points/saga.ts#L372). the subsequent config updates affect the consistency of config data, but aren't preventing us from proceeding.